tests/base_new_framework.py

```diff
@@ -0,0 +1,200 @@
+import backoff
```

Comment:
I'm not up to speed on why we need two base files when transitioning to the new framework.  Maybe you and Harrison can get me up to speed?

Reply:
Most of the methods that are present in the original Base file are moved to Base_case in Tap-tester, if we want a single base file then we will have to migrate all the other tests to the new framework.
Currently our main focus is on implementing the missing tests using the new frame work once that is done. We might then work on the other tests and just maintain one single base file.